src/CommonDropdown.php

```diff
@@ -202,22 +202,31 @@ public function defineTabs($options = [])
         return $ong;
     }
 
-
-    final public function displayHeader()
+    public function displayHeader()
     {
-
-        if (empty($this->third_level_menu)) {
-            $this->third_level_menu = $this->getType();
-        }
-        Html::header(
-            $this->getTypeName(Session::getPluralNumber()),
-            '',
-            $this->first_level_menu,
-            $this->second_level_menu,
-            $this->third_level_menu
+        Toolbox::deprecated(
```

Comment:
Method logic should not be removed when it is deprecated.

Reply:
Indeed, I forgot to call the new method... should be fixed now